docs/SRS-Volere/SRS.tex

```diff
@@ -677,8 +677,13 @@ \section*{Appendix --- Reflection}
 
   \medskip
   \textbf{Jason Nam}\\
-  \textit{Chosen Skills}: \\
-  \lips
+  \textit{Chosen Skills: (a), (c), (g)}\\
+  During the early stages of project team formation and idea generation, I was tasked with the job of undertaking feasibility studies related to implementing OCR for the team's project solution. At that point in time, I had extensive experience working with Machine Learning for Natural Language Processing. However, I had never had the opportunity to learn proper Machine Learning models related to OCR. I am eager to learn the workings behind OCR and how we can implement it in our project.
+  Similar to my previous point, I also had the opportunity to experience full-stack development involving web development. However, I never had a chance to learn Mobile App development, especially front-end app development. I am confident I can utilize my skills from web development to transfer over to app development and improve from there on.
+  Finally, I want a tangible skill I can acquire during the duration of this project. I believe that skills related to software testing and coverage will be very valuable later on in life. \par
+  My approach to acquiring OCR and Machine Learning Recognition skills will involve extensively reading over user manual for Tesseract OCR package. Fortunately, there are many sources of information related to Tesseract OCR online in many different platforms, including YouTube, Tesseract Github Repository, and official user manuals. 
```

Comment:
- My approach to acquiring OCR and Machine Learning Recognition skills will involve extensively reading over **the** user manual for Tesseract OCR package.
- Please also use \textit{Tesseract-OCR} and \textit{YouTube} for consistency